PATCH-P1-FIX-UPSTREAM: Reduce the number of rounds and operations to avoid testcase timeout.
BUGS: bnc#937343, MDEV-8443

Maintainer: Kristyna Streitova <kstreitova@suse.com>

--- a/mysql-test/suite/innodb/r/innodb_simulate_comp_failures.result	
+++ a/mysql-test/suite/innodb/r/innodb_simulate_comp_failures.result	
@@ -5,4 +5,4 @@ CREATE TABLE t1(id INT AUTO_INCREMENT PRIMARY KEY, msg VARCHAR(255), KEY msg_i(m
 SET GLOBAL innodb_simulate_comp_failures = 25;
 SELECT COUNT(*) FROM t1;
 COUNT(*)
-10000
+2000
--- a/mysql-test/suite/innodb/t/innodb_simulate_comp_failures.test	
+++ a/mysql-test/suite/innodb/t/innodb_simulate_comp_failures.test	
@@ -1,8 +1,8 @@ 
 --source include/big_test.inc
 # test takes too long with valgrind
 --source include/not_valgrind.inc
---let $num_inserts = 10000
---let $num_ops = 10000
+--let $num_inserts = 2000
+--let $num_ops = 4000
 --source suite/innodb/include/innodb_simulate_comp_failures.inc
 # clean exit
 --exit
